source/core/src/main/com/csse3200/game/components/player/PlayerCoinDisplay.java

```diff
@@ -63,6 +69,31 @@ private void updateCoinLabel() {
         coinLabel.setText("Coins: x" + coinsComponent.getCoins());
     }
 
+    /**
+     * When a player successfully defeats an animal, the
+     * amount of coins earned is displayed on the screen for few seconds
+     *
+     * @param coins the amount of coins an animal gave from defeating
+     */
+    private void showCoins(int coins) {
+        ServiceLocator.getResourceService().playSound("sounds/collectCoin.mp3");
+        String text = String.format("+%d coins added", coins);
+        Label coinText = new Label(text, skin, "small");
+        Table table = new Table();
+        table.top().padTop(150f);
+        table.setFillParent(true);
+        table.row().padTop(5f);
+        table.add(coinText).padTop(5f);
+        stage.addActor(table);
+        // unrender the label after 3 second of display
+        Timer.schedule(new Timer.Task() {
+            @Override
+            public void run() {
+                coinText.remove();
+            }
+        }, 1f);
```

Comment:
3 seconds or 1 second?

Reply:
3 seconds ends up being too long

Reply:
check it out with 3 seconds and if you prefer that, i can change it